layers/shaders/memory_decompression/readme.txt

```diff
@@ -0,0 +1,2 @@
+Use compile.bat/compile.sh to generate SPIRV for the GLSL shaders used in memory_decompression

```

Comment:
When would it be necessary to regenerate these files?  Please document it so someone in the future will know to do it.